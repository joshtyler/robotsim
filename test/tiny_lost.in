0 0
0 0 N
F
0 0 N
F
