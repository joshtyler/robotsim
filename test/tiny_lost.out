0 0 N LOST
0 0 N
